code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm

```diff
@@ -0,0 +1,394 @@
+	price = 100
+	next = new/datum/malf_research_ability/synthetic/infect_apc()
+	name = "Reset Cyborg Module"
+
+
+/datum/malf_research_ability/synthetic/infect_apc
+	ability = new/datum/game_mode/malfunction/verb/infect_apc()
+	price = 700
+	next = new/datum/malf_research_ability/synthetic/overclock_borg()
+	name = "Infect APC"
+
+
+/datum/malf_research_ability/synthetic/overclock_borg
+	ability = new/datum/game_mode/malfunction/verb/overclock_borg()
+	price = 1300
+	next = new/datum/malf_research_ability/synthetic/synthetic_takeover
+	name = "Overclock Cyborg"
+
+
+/datum/malf_research_ability/synthetic/synthetic_takeover
+	ability = new/datum/game_mode/malfunction/verb/synthetic_takeover()
+	price = 5000
+	name = "Synthetic Takeover"
+
+// END RESEARCH DATUMS
+// BEGIN ABILITY VERBS
+
+/datum/game_mode/malfunction/verb/reset_module(var/mob/living/silicon/robot/target = null as mob in get_linked_cyborgs(usr))
+	set name = "Reset Cyborg Module"
+	set desc = "100 CPU - Forces a module reset on a cyborg that is enslaved to you. Has a chance of failing."
+	set category = "Software"
+	var/price = 100
+	var/mob/living/silicon/ai/user = usr
+	if(!ability_prechecks(user, price))
+		return
+
+	if(target && !istype(target))
+		user << "This is not a cyborg."
+		return
+
+	if(target && target.connected_ai && (target.connected_ai != user))
+		user << "This cyborg is not connected to you."
+		return
+
+	if(!target)
+		var/list/robots = list()
+		var/list/robot_names = list()
+		for(var/mob/living/silicon/robot/R in silicon_mob_list)
+			if(istype(R, /mob/living/silicon/robot/drone))	// No drones.
+				continue
+			if(R.connected_ai != user)						// No robots linked to other AIs
+				continue
+			robots += R
+			robot_names += R.name
+
+
+		var/targetname = input("Select reset target: ") in robot_names
+		for(var/mob/living/silicon/robot/R in robots)
+			if(targetname == R.name)
+				target = R
+				break
+
+	if(target)
+		if(alert(user, "Really try to reset cyborg [target.name]?", "Reset Cyborg", "Yes", "No") != "Yes")
+			return
+		if(!ability_pay(user, price))
+			return
+		user.hacking = 1
+		user << "Attempting to reset the cyborg. This will take approximately 20 seconds."
+		sleep(200)
+		if(prob(30))
+			user <<"Reset attempt failed!"
+			return
+		if(target)
+			user << "Successfully sent reset signal to cyborg.."
+			target << "Reset signal received.."
+			sleep(20)
+			user << "Cyborg reset."
+			target << "You have had your module reset."
+			log_ability_use(user, "reset cyborg", target)
+			target.uneq_all()
+			target.modtype = initial(target.modtype)
+			target.hands.icon_state = initial(target.hands.icon_state)
+			target.module.Reset(target)
+			qdel(target.module)
+			target.module = null
+			target.updatename("Default")
+		else
+			user << "Unable to reset cyborg."
+
+		user.hacking = 0
+
+
+
+
+/datum/game_mode/malfunction/verb/infect_apc(obj/machinery/power/apc/A as obj in get_apcs(src))

```

Comment:
`get_apcs(src)` <-- this should be `get_apcs()` because `get_apcs` doesn't take any arguments.